assignments/objects.js

```diff
@@ -18,27 +18,70 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+  "id": 01,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
 
+const kennan = {
+  "id": 02,
+  "name": "Kennan",
+  "email": "kdiben1@dtinypic.com",
+  "gender": "M",
+  speak() {
+    console.log("Hello, my name is Kennan!")
+  }
+}
+
+const keven = {
+  "id": 03,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
 
+const gannie = {
+  "id": 04,
+  "name": "Gannie",
+  "email": "gmartinsen3@illinois.edu",
+  "gender": "F"
+}
+
+const antonietta = {
+  "id": 05,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  multiplyNums(num1, num2) {
+    return num1 * num2
```

Comment:
Great job adding the function here to the `antonietta` object here. It could also be done using the Object dot notation which can be used to modify or add a new property of field as such:

`antonietta.mutltiplyNums( (num1, num2) => num1 * num2)` outside the scode of this objec (so anywhere past @60)